tests/rt.conf.restart

```diff
@@ -0,0 +1,3 @@
+COMPILE | coupledFV3_CCPP_MOM6_CICE_CMEPS           |   | hera.intel  | fv3 |
```

Comment:
The restart test is awkward. Since it doesn't generate a baseline, it is only run against current or new baselines. And the 3d baseline must have already been created. It would be good if there could be a conditional in rt.sh---run the restart test only after running the 2d and 3d tests and not if you're creating baselines. 

I have this rt.conf.restart just as a convenience so that I can cut/paste the restart run line into the rt.conf for the final comparison. I could remove the file or rename it.

Reply:
I will remove this file.